specifications/api/swagger.yaml

```diff
@@ -4124,6 +4124,17 @@ x-errMalformedTxPayload: &errMalformedTxPayload
       type: string
       enum: ['malformed_tx_payload']
 
+x-errMempoolIsFull: &errMempoolIsFull
+  <<: *responsesErr
+  title: malformed_tx_payload
+  properties:
+    message:
+      type: string
+      description: May occur when submitting a serialized transaction to an external service with full mempool.
+    code:
+      type: string
+      enum: ['mempool_is_full']
```

Comment:
What is the point of this `code` field esp. if enum is a singleton?

Reply:
I'm not sure how to fix this. I will dig

Reply:
It seems we are doing this everywhere to say they the code field can have only a subset of vaules

Reply:
FYI https://github.com/OAI/OpenAPI-Specification/issues/1313